File: Programar em C#/16-Ciclo-for.cs
// Programar em C# #16 - Ciclo for

using System;

namespace Base {
    class CicloFor {
        public void Run() {
            string[] materialEscolar = new string[6] {
                "Mochila", // 0
                "Estojo", // 1
                "Lápis", // 2
                "Borracha", // 3
                "Afia", // 4
                "Tesoura" // 5
            };

            /* Console.WriteLine(materialEscolar[0]);
            Console.WriteLine(materialEscolar[1]); */

            for (int indice = 0; indice < materialEscolar.Length; indice++)
                Console.WriteLine($"materialEscolar[{indice}]: {materialEscolar[indice]}");
        }
    }
}

File: Programar em Java/16-Ciclo-for.java
// Programar em Java #16 - Ciclo for

package com.caffeinealgorithm.programaremjava;

public class CicloFor {
    public void Run() {
        String[] materialEscolar = new String[] {
                "Mochila",  // 0
                "Estojo",   // 1
                "Lápis",    // 2
                "Borracha", // 3
                "Afia",     // 4
                "Tesoura"   // 5
        };

        /* System.out.println(materialEscolar[0]);
        System.out.println(materialEscolar[1]); */

        for (int indice = 0; indice < materialEscolar.length; indice++)
            System.out.println(new StringBuilder().append("materialEscolar[").append(indice).append("]: ").append(materialEscolar[indice]));
    }
}